lib/openstack_query/handlers/server_side_handler.py

```diff
@@ -95,12 +101,12 @@ def _check_filter_mapping(
         :param filter_params: a dictionary of params to check if valid for filter func
         """
         try:
-            logger.debug(
-                "checking server-side filter function against provided parameters\n\t%s",
-                "\n\t".join(
-                    [f"{key}: '{value}'" for key, value in filter_params.items()]
-                ),
-            )
+            # logger.debug(
+            #     "checking server-side filter function against provided parameters\n\t%s",
+            #     "\n\t".join(
+            #         [f"{key}: '{value}'" for key, value in filter_params.items()]
+            #     ),
+            # )
```

Comment:
Just so we don't forget this, we should add this back or resolve the issues we were seeing with None types